lib/compress/zstd_compress.c

```diff
@@ -1355,6 +1363,10 @@ static size_t ZSTD_estimateCCtxSize_usingCCtxParams_internal(
 
     size_t const cctxSpace = isStatic ? ZSTD_cwksp_alloc_size(sizeof(ZSTD_CCtx)) : 0;
 
+    /* Require two allocations for alignment to 64 bytes, call ZSTD_cwksp_alloc_size() twice. */
+    size_t const alignSpace = ZSTD_cwksp_alloc_size(ZSTD_CWKSP_ALIGN_TABLES_BYTES/2)
+                            + ZSTD_cwksp_alloc_size(ZSTD_CWKSP_ALIGN_TABLES_BYTES/2);
```

Comment:
I think the alignment ptr/size logic should be moved to `zstd_ckwsp.h`. It can report the amount of space it needs for alignment.